{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/globals/edit-bar.css

```diff
@@ -77,9 +82,9 @@
 */
 .pg-EditBar_FrameContainer {
   position: fixed;
-  top: 36px;
+  top: 0;
   right: 0;
-  bottom: 0;
+  bottom: 36px;
```

Comment:
Where is 36 from?

Reply:
24 (typical line height) + 6 px (0.25vr top padding) + 6px (0.25vr bottom padding). I'll 1) remove the 0.25 vr and replace it with '6px' (the styleguide says not to use very small fractional `vr`) 2) clarify this with a comment 3)  change the 40px used in `base/layout.css` which contradicts this.